frontend/Dockerfile

```diff
@@ -1,7 +1,7 @@
 
 # Stage 1: Compile and Build angular codebase
 # Use official node image as the base image
-FROM node:16 as build
+FROM node:20 as build
```

Comment:
I've some problems building the container with node:20.
The container gets stuck at the "npm ci" step. I had to switch the image to node:20-alpine, and then the installation worked without any issues. Is there anything against switching to alpine for that particular image? I don't see any problem with that.